### PR TITLE
Products/Projects title added to navigation bar

### LINKED ISSUE #5054: Add Projects in navigation bar
### Current Behavior
We list Products and Projects inside the same dropdown in the navigation bar.
image

### Desired Behavior
1. Change **_Products_** title in the navigation bar to **_Products/Projects_**.
image

2. Add a **_Project_** title inside the dropdown.
3. List all products inside **_Products_** title  and projects inside **_Projects_** title. 
    - All list item above "---" in the current implementation is Product and list items below "---" are projects 
5. Link **_Projects_** title to [layer5.io/projects](https://layer5.io/projects) page

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by aakashdeepsil**
**Description**

This PR fixes #5054 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 98f026e8351369721bf88e7631824d6f9629c66a at: https://654673d062392e28cd9ab5e4--layer5.netlify.app

File: src/sections/General/Navigation/utility/menu-items.js [hunk 1]
```diff
@@ -80,9 +80,32 @@ const Data = {
...
       path: "/projects",
       subItems: [
+        // {
+        //   name: "Image Hub",
+        //   path: "/projects/image-hub",
+        //   sepLine: true
+        // },
+        {
+          name: "PRODUCTS",
+          path: "/products",
```
Comment (sudhanshutech) [inline src/sections/General/Navigation/utility/menu-items.js:93]: @aakashdeepsil can you navigate this link to meshmap for now, until we have products page up. This link is broken

Comment (sudhanshutech) [inline src/sections/General/Navigation/utility/menu-items.js:93]: @aakashdeepsil can you navigate to meshmap here until products page is up. This is broken

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by balagowda** [comment]
can it be better in this way[

](url)

**Comment by balagowda** [comment]
can it be more productive in this way??

**Comment by balagowda** [comment]
@sudhanshutech , can this be more friendly??

File: src/sections/General/Navigation/utility/menu-items.js (hunk 1 above)
Comment (aakashdeepsil) [reply src/sections/General/Navigation/utility/menu-items.js:93]: Sure I will make this change.

**Comment by l5io** [comment]
🚀 Preview for commit 537dd844511adef014f11de3c63e96d031ea6025 at: https://6546c67910f7c9602e1f0e57--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]
LGTM, thanks @balagowda for this but this looks good and lets get you assigned to other issues

**Comment by abhijeetgauravm** [comment]
> @sudhanshutech , I have made the below changes, would it be better to integrate it with this? Screenshot 2023-11-04 at 10 03 15 PM

@balagowda Here you'll find many good-first and help-wanted issues:
https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22help+wanted%22+

Feel free to check these out and comment to get assigned.

**Pull request merged**